state/src/style.rs

```diff
@@ -90,7 +90,14 @@ impl State<CustomAttributeValues> for Style {
                     }
                     "shadow" => {
                         if let Some(value) = attr.value.as_text() {
-                            style.shadows = parse_shadows(value, *scale_factor)
+                            style.shadows = split_shadows(value)
```

Comment:
Why was `parse_shadows` removed?

Reply:
I renamed it to `split_shadows` and split off the parsing logic so that the splitting could be shared in between `TextShadow` and `Shadow`, which each have their own parsers.